monster-cards/src/widgets/card/card-sprite-large.styl

```diff
@@ -0,0 +1,147 @@
+&.Cards-Momentum14 {
+	background-position: -0px -0px;
+}
```

Comment:
If the sprite sheet contained images in proper order, e.g. 1 - 32, this could be simplified by using iteration:

```stylus
for num in (1..34) {
	.Cards-Mayhem{num} {
		background-position: -0px (-270px * num);
	}
}
```